imapautofiler/tests/base.py

```diff
@@ -81,3 +89,11 @@ def i18n_msg(self):
                 construct_message(I18N_MESSAGE)
             )
         return self._msg
+
+    @property
+    def recent_msg(self):
+        if self._recent_msg is None:
+            self._recent_msg = email.parser.Parser().parsestr(
+                construct_message(RECENT_MESSAGE)
+            )
+        return self.recent_msg
```

Comment:
This is going to recurse because it's using the property name not the internal attribute name.